src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/events/PluginPatchsetCreatedEvent.java

```diff
@@ -47,29 +48,83 @@
     private boolean excludeDrafts = false;
     private boolean excludeTrivialRebase = false;
     private boolean excludeNoCodeChange = false;
+    private boolean excludePrivateState = false;
+    private boolean excludeWipState = false;
 
     /**
      * Default constructor.
      */
+    @DataBoundConstructor
     public PluginPatchsetCreatedEvent() {
-        this(false, false, false);
+        this(false, false, false, false, false);
     }
 
     /**
      * Standard DataBoundConstructor.
      * @param excludeDrafts if drafts should be excluded or not.
      * @param excludeTrivialRebase if trivial rebases should be excluded or not.
      * @param excludeNoCodeChange if message-only changes should be excluded.
+     * @param excludePrivateState if private state changes should be excluded.
+     * @param excludeWipState if wip state changes should be excluded.
      */
-    @DataBoundConstructor
+    @Deprecated
     public PluginPatchsetCreatedEvent(boolean excludeDrafts,
         boolean excludeTrivialRebase,
-        boolean excludeNoCodeChange) {
+        boolean excludeNoCodeChange,
+        boolean excludePrivateState,
```

Comment:
By keep the original signature with deprecation I meant the original signature before you started making changes :)
We try to strive for keeping binary copatibility between releases of the plugin (since there is no clear way of indicating defined APIs)
And this change breaks binary compatibility because the signature `PluginPatchsetCreatedEvent(boolean, boolean)` is gone.

Reply:
Sorry for the misunderstanding, I should aware that. 
If you still need to change anything, just inform me. 
Thanks for the patience.